### PR TITLE
running notebook with custom loss class for skorch.

### PR DESCRIPTION
**Pull request by t-kimber**
## Description
This PR focuses on how to use skorch (Pytorch + sklearn) package for training the machine learning models.
Advantages:
Cool features pre-implemented, e.g.
- Built in cross-validation
- early stopping
- callbacks
- etc. 

## Todos
- [ ] Include skorch in API
- [x] running example
  - [x] custom loss class for pIC50 to enable observation model

## Questions
- [ ] Question1

## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by jaimergp** [comment]
This is so cool!
